### DISCUSSION TITLE
Bug: `:tada:` shortcode no longer resolves to 🎉 emoji in issue comments posted via the API

### DISCUSSION THREAD

**Discussion by franky47** [community/community#78334] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

The Markdown renderer seems to have stopped rendering the `:tada:` shortcode into the associated 🎉, for programmatic text published via the API.

Messages posted with this shortcode from the semantic-release GitHub Actions bot used to show this emoji, but now only show the shortcode in plain text ([example](https://github.com/47ng/next-usequerystate/pull/415#issuecomment-1846309757)). It's not a regression in the bot code, as messages posted way back (that did show the emoji correctly) now show the shortcode.

The issue happens on comments on both issues and pull requests.

Typing the `:tada:` shortcode in the markdown editor on github.com and GitHub mobile both resolve correctly to the emoji, this issue happens only when the comment is published via the API. I ran a test using the GraphQL API to post a comment, and it replicated the issue: https://github.com/franky47/actions-laboratory/issues/14

Edit: I was able to reproduce the issue in my test thread with other emoji shortcodes (`:rocket:` and `:package:`), however those still display as emoji for messages posted by the semantic-action bot.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by whaagmans** [comment]
This is also the case for shortcodes used in commits. Unless this functionality has been removed. It was possible before to put shortcodes in commits which would show in the commits and repository page. This stopped working around a month or 2 ago.

**Comment by bewuethr** [comment]
It looks like it's broken for emoji shortcodes in releases, too:

![image](https://github.com/community/community/assets/8521043/63a04df4-117d-4862-9082-c04483cff643)

**Reply by bewuethr** [reply]
Notably, `:octocat:` still works.

**Comment by franky47** [comment, marked as answer]
It looks like the situation has been resolved. :tada:
